packages/mds-audit/api.ts

```diff
@@ -188,12 +191,16 @@ function api(app: express.Express): express.Express {
             })
 
             // Create the audit start event
+            const provider_event = await readEvent(provider_device_id)
```

Comment:
In general, we don't use snake case when we can avoid it. `providerEvent` would be preferable.

Reply:
Updated my new variables to use camelCase. It could be fun to run an autoformatter on the repo at some point, snake_case is pretty pervasive.

Reply:
We already use `Prettier/ESLint`, but we don't have any casing rules enabled. Generally speaking, we prefer camelCase unless it is a db column, query string parameter, request body (as dictated by some spec), etc.